src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -1123,20 +1129,18 @@ class ApplicationManagerImpl
   void RemoveAppFromTTSGlobalPropertiesList(const uint32_t app_id) OVERRIDE;
 
   /**
-   * DEPRECATED
    * @brief method adds application in FULL and LIMITED state
    * to on_phone_call_app_list_.
    * Also OnHMIStateNotification with BACKGROUND state sent for these apps
    */
-  void CreatePhoneCallAppList();
+  DEPRECATED void CreatePhoneCallAppList();
 
   /**
-   * DEPRECATED
    * @brief method removes application from on_phone_call_app_list_.
    *
    * Also OnHMIStateNotification with previous HMI state sent for these apps
    */
-  void ResetPhoneCallAppList();
+  DEPRECATED void ResetPhoneCallAppList();
 
   // TODO(AOleynik): Temporary added, to fix build. Should be reworked.
```

Comment:
is it still Temporary  work?

Reply:
@yang1070, no, this is a comment from 2014 according to git history. For now all required rework were done, but this function is still needed due to new functionality appeared in application manager during last years. I removed that comment to avoid possible confusions in future. See https://github.com/smartdevicelink/sdl_core/pull/1820/commits/74837cf41e96439eb131227c961405b3af5091bb